### ISSUE TITLE
Disk limit quickly reached on c9

### ISSUE THREAD

**Issue by mizujin** [FreeCodeCamp/FreeCodeCamp#2751]
I've been struggling with this issue. Right after setting everything for the basejumps, I can't do anything as c9 seems to load infinitely and telling that max disk space has been reached. 

I've tried to set a local dev environment but i'm not yet comfortable with this.

Could we use an alternative, lighter generator that could work with the basejumps ? Someone mentionned Clementine.

**Comment by 1hella** [comment]
This is a big problem. I haven't seen anybody that has been able to keep moving forward with cloud9 and the angular-fullstack generator.

**Comment by benschenker** [comment]
Yeah this seems pretty bad, just tried going through it and it wasn't pretty (nor did it work in the end).  Any ideas @QuincyLarson?

**Comment by awesomeaniruddh** [comment]
What about koding.com and nitrous.io?

koding.com has 3GB of disk space and 1 gigabyte of RAM on the free plan.

**Comment by awesomeaniruddh** [comment]
This *should* be resolved when Clementine JS comes out.

**Comment by ybentz** [comment]
Hey guys,
I've also been struggling with the setup on cloud9 and the RAM issue so I started looking for alternatives, I nitrous.io, very user friendly just like cloud9 but it gives you 1GB RAM on the free package and it also comes with node which is sweet.
Ran through the whole process with no memory problems but when I tried to run the mongodb command I discovered that it doesn't come with mongo out of the box so I installed it according to the instructions on their site but I had to install mongo3 because 2 isn't available via apt-get anymore.
Then when I try to run the mongod command again I get this error:
```
Error parsing command line: adjacent parameter is empty in 'bind_ip'
```
So I tried removing the '=' to match the instructions on the mongo website and now I get this:
```
2015-10-10T00:34:23.718+0000 I CONTROL  ** WARNING: --rest is specified without --httpinterface,
2015-10-10T00:34:23.718+0000 I CONTROL  **          enabling http interface
Invalid command: data
```
and the help opens.
I tried playing with the order of the parameters in the command and also omit some of them and update the /etc/mongod.conf file but with no luck, every time I get a different error and I can't figure it out.

Does anyone have an idea of how to run it?
Thanks!
